lib/checkPullRequestReview.js

```diff
@@ -381,9 +382,63 @@ const handleResponseToReview = async (context) => {
   }
 };
 
+/**
+ * @param {import('probot').Context} context
+ * @param {string[]} assignees
+ * @param {string} comment
+ * @param {import('probot').Octokit.PullsGetResponse} pullRequest
+ */
+
+//This is a function to check if there are more than 24 Hours with the Pr
+const isMaxReviewPeriodExceeded = async (review) => {
+  let timeOfReviewSubmission = review.submitted_at;
+  timeOfReviewSubmission = new Date(timeOfReviewSubmission);
+  let currentTime = new Date.now();
+  let diff = Math.abs(timeOfReviewSubmission - currentTime);
+  return diff > MAX_REVIEW_PERIOD;
+};
+
+const checkForReviewersWithPendingReview = async(context) => {
+  const allOpenPullRequests = await getAllOpenPullRequests(context);
+  await allOpenPullRequests.map(
+    async (pullRequest) => {
+      const { data: allReviews } = await context.github.pulls.listReviews(
+        context.repo({
+          pull_number: pullRequest.number,
+        })
+      );
+
+      const allReviewers = [];
+      allReviews.map((review) => {
+        const booleanVariable = isMaxReviewPeriodExceeded(review);
+        // If Max Period Exceeded then We need to ping that reviewer.
+        if (booleanVariable) {
+          const reviewer = review.user.login;
+          if (!allReviewers.includes(reviewer)) {
+            allReviewers.push(reviewer);
+          }
```

Comment:
Should you also check if the reviewer is assigned to the pull request?

Reply:
Rethinking the issue, I think it will be safer and easier check assigned reviewers and then check when their last review was, if any, and decide to ping them if they fit the 24 hours category.

Reply:
Sir, I think this will not be a problem, As soon code owners will be decided by CODEOWNERS File. So this might become redundant.
Looking forward to your views on this.